randutil/crypto_test.go

```diff
@@ -45,3 +48,79 @@ func (s *cryptoRandutilSuite) TestCryptoToken(c *C) {
 	c.Assert(err, IsNil)
 	c.Check(b, HasLen, 5)
 }
+
+var (
+	KernelTestUUID        = "1031319a-b661-4c01-aafa-6def8a118944"
+	KernelUUIDPathDefault = randutil.KernelUUIDPath
+)
+
+func (s *cryptoRandutilSuite) TestRandomKernelUUIDNoFile(c *C) {
+	tmpDir := c.MkDir()
+	uuidPath := filepath.Join(tmpDir, "no-file")
+
+	randutil.KernelUUIDPath = uuidPath
```

Comment:
(nipitck^3) given that this is used quite a bit and that it would make sense to unexport KernelUUIDPath a `MockKernelUUIDPath(newPath string) (*restore func())` would make this setup slightly shorter.